<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frmrecepcion_pedidos</class>
 <widget class="QDialog" name="Frmrecepcion_pedidos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recepci칩n de pedidos</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_36">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>27</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);
background-color: #304163;</string>
     </property>
     <property name="text">
      <string>Recepci칩n de Pedidos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_22">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>27</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 127);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>AREA DE COMPRAS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Proveedor:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="txtproveedor"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnBuscaProv">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../maya.qrc">
             <normaloff>:/Icons/PNG/search.png</normaloff>:/Icons/PNG/search.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Realizados entre:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateEdit" name="txtFecha_ini">
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="chktodos">
         <property name="text">
          <string>Todos</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="txtFechaFin">
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="chkpendientes">
         <property name="text">
          <string>Pendientes</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QRadioButton" name="chkcompletados">
         <property name="text">
          <string>Completados/cerrados</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QTableView" name="tablaPedidos">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="btnBuscar">
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>27</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #304163;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Filtrar Pedidos</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <widget class="QPushButton" name="btalbaran">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Crear albar치n</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btnFactura">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Crear Factura</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QTableWidget" name="tablaLineas">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>1</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btnImprimir">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Imprimir</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblAlbaran">
           <property name="text">
            <string>Albar치n Num:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtAlbaran">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblfecha_albaran">
           <property name="text">
            <string>Fecha:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="txtFecha_albaran">
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblfactura">
           <property name="text">
            <string>Factura Num:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtFactura">
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblFecha_factura">
           <property name="text">
            <string>Fecha:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="txtFecha_factura">
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkForzarCierre">
           <property name="text">
            <string>Forzar Cerrar Pedido</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btetiquetas">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Etiquetas</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtFecha_ini</tabstop>
  <tabstop>txtFechaFin</tabstop>
  <tabstop>chkcompletados</tabstop>
  <tabstop>tablaPedidos</tabstop>
  <tabstop>btalbaran</tabstop>
  <tabstop>btnFactura</tabstop>
  <tabstop>btnImprimir</tabstop>
  <tabstop>btetiquetas</tabstop>
 </tabstops>
 <resources>
  <include location="../maya.qrc"/>
 </resources>
 <connections/>
</ui>
